src/main/resources/static/templates/XXEVulnerability/LEVEL_1/XXE.js

```diff
@@ -14,33 +7,35 @@ let buildXmlRequest = function (
 ) {
   let xmlRequestDocument = document.implementation.createDocument("", "", null);
   let bookElement = xmlRequestDocument.createElement("book");
+  
   let bookNameElement = xmlRequestDocument.createElement("name");
-  bookNameElement.innerHTML = bookName;
+  bookNameElement.textContent = bookName; // Safe: textContent is used instead of innerHTML
   bookElement.appendChild(bookNameElement);
 
   let authorElement = xmlRequestDocument.createElement("author");
-  authorElement.innerHTML = author;
+  authorElement.textContent = author; // Safe: textContent is used instead of innerHTML
   bookElement.appendChild(authorElement);
 
   let isbnElement = xmlRequestDocument.createElement("isbn");
-  isbnElement.innerHTML = isbn;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities